client/packages/programs/src/JsonForms/common/hooks/useDebouncedTextInput.ts

```diff
@@ -49,8 +48,8 @@ export const useDebouncedTextInput = (
   // performance issue
   const debouncedText = useDebouncedValue(text, 500);
   useEffect(() => {
-    handleChange(error ? undefined : debouncedText);
-  }, [error, debouncedText]);
+    handleChange(debouncedText);
+  }, [handleChange, debouncedText]);
```

Comment:
I think the `handleChange` in the dependency array here is a mistake. Seems to be getting into an infinite loop, and not actually saving anything.

When I remove `handleChange` here, it seems to work okay. Was there a particular reason why you included it?

Reply:
thanks I removed it and this fixed it